packages/create-block-app/create-block-app.js

```diff
@@ -157,7 +157,18 @@ const usage = commandLineUsage(helpSections);
     await fs.writeJson(blockMetadataPath, blockMetadata, { spaces: 2 });
   }
 
+  const chalk = (await import("chalk")).default;
+
+  console.log(
+    chalk.bold("-".repeat(Math.min(process.stdout.columns ?? 48, 48))),
+  );
   console.log(
-    `Your ${blockName} block is ready to code in ${resolvedBlockPath}.\nRun 'yarn install && yarn dev' to get started`,
+    `Your ${chalk.bold(blockName)} block is ready to code in ${chalk.bold(
+      resolvedBlockPath,
+    )}.\n` +
+      `Run ${chalk.blue(`cd ${resolvedBlockPath}`)} and then \n` +
```

Comment:
If `resolvedBlockPath` contains spaces, the command will end up broken. Let’s escape this value: https://github.com/nodejs/node/issues/34840#issuecomment-676398171

Reply:
Optional (not sure in this). We can also use [`path.relative`](https://nodejs.org/api/path.html#pathrelativefrom-to) in `cd` command to make it shorter. It will be just `./block-subpath` in most cases.

Reply:
Both done in https://github.com/blockprotocol/blockprotocol/pull/603/commits/cd1b786aa46d7cd98341c96b12de6a9d4ace44d8